cmd/algod/main.go

```diff
@@ -369,6 +370,30 @@ func run() int {
 	return 0
 }
 
+var startupConfigCheckFields = []string{
```

Comment:
Consider reporting all overrides so that we don't have to keep updating this in the future (or forgetting to).

Reply:
I would prefer to keep it very limited and pre-defined to start, because there are a lot of various ways we automatically change config values at startup time, and there are also config vars that are private or not interesting/useful to know if they were changed.

Reply:
Having to remember later on what we do/don't care about can brittle.

Reply:
oh a struct tag would be a good idea!